web/themes/custom/votegov/src/sass/components/quick-links.scss

```diff
@@ -0,0 +1,63 @@
+@use "uswds-core" as *;
+@use "variables" as *;
+
+.vote-quicklinks {
+  background-color: #11385B;
+  color: #FFF;
+  @include u-padding-x(3);
+  @include u-padding-y(6); 
+}
+
+.vote-quicklinks__container {
+  @include grid-container;
+  @include grid-row;
+  @include u-flex('justify');
+
+  @include at-media('tablet-lg') {
+      @include u-padding-y(9);
+  }
+}
+
+.vote-quicklinks__heading {
+  padding-bottom: 20px;
+  
+  @include at-media('tablet-lg') {
+    @include grid-col(4);
+    @include u-order(1);
+    @include u-padding-y(2);
+    padding-left: 30px;
+  }
+
+    h2 {
+      color: #FFF;
+    }
+}
+
+.vote-quicklinks__red-line {
+    border-bottom: 4px solid #C74345;
+    width: 70%;
+
+    @include at-media('tablet-lg') {
+      @include grid-col(2);
+      @include u-order(2);
+      transform: rotate(90deg);
+      margin: auto;
+  }
+}
+
+.vote-quicklinks__content{
+    padding-top: 20px;
+
+  @include at-media('tablet-lg') {
+    @include grid-col(6);
+    @include u-order(3);
+}
+
+    p {
+      padding-bottom: 20px;
+    }
+
+    a {
+      color: #0AAAE2
```

Comment:
There is a `$ac-light-cool` color that you can use after you uncomment it in the variables file.

Reply:
Add the $ac-light-cool to links and uncommented from variables file